cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -70,6 +150,7 @@ type devicePlugin struct {
 	sysfsDir string
 	devfsDir string
 
+	policy  preferredAllocationPolicyFunc
```

Comment:
if the plugin is restarted with a new policy, the existing allocations remain. Is that OK?

Is there a use case where the policy could be node specific?

Reply:
It true that if restarting the plugin with a new policy, the existing allocations remain with old policy. But I think it is okay because user should know about this and what to do should be in upper level(maybe re-apply the pod in k8s).

Reply:
I am a little confused about what the meaning of "Is there a use case where the policy could be node specific"? Could you explain more, thanks.
My understand is there are many GPU nodes and the preferred allocation method is based on SharedDevNum >1, then the policy is meaningful for allocating devices among these GPU nodes.

Reply:
> It is true that if restarting the plugin with a new policy, the existing allocations remain with old policy.

how about adding a comment about this next to the `policy` field in `devicePlugin` struct

Reply:
Exactly, it is better to add a note here.